examples/src/main/scala/examples/commons/PublicKey25519NoncedBox.scala

```diff
@@ -39,19 +33,21 @@ object PublicKey25519NoncedBox extends ScorexEncoding {
     ).asJson
 }
 
-object PublicKey25519NoncedBoxSerializer extends Serializer[PublicKey25519NoncedBox] {
+object PublicKey25519NoncedBoxSerializer extends ScorexSerializer[PublicKey25519NoncedBox] {
 
-  override def toBytes(obj: PublicKey25519NoncedBox): Array[Byte] =
-    obj.proposition.pubKeyBytes ++
-      Longs.toByteArray(obj.nonce) ++
-      Longs.toByteArray(obj.value)
 
+  override def serialize(obj: PublicKey25519NoncedBox, w: Writer): Unit = {
+    PublicKey25519PropositionSerializer.serialize(obj.proposition, w)
+    w.putULong(obj.nonce)
```

Comment:
It seems the nonce could be negative (it is generated by applying Longs.fromByteArray to pseudorandom bytes).